app/server/utils/rewards/queries.py

```diff
@@ -0,0 +1,52 @@
+from datetime import datetime
+from datetime import timedelta
+
+from server import db
+
+
+def users_with_corresponding_total_outward_txs(created_since: int,
```

Comment:
Would it not be better to integrate with `app/server/model/transfer_account.py` here instead? it already has a list of credit transfer as a relation.

Reply:
I tried running a scenario with this, I don't think there's a change in overhead @nolash but I am curios what did you have in mind with this suggestion ?